README.md

```diff
@@ -122,6 +122,15 @@ result = whisper.decode(model, mel, options)
 print(result.text)
 ```
 
+
+## Extensions and integrations
+
+A list of community-contributed extended functionality and integrations with whisper.
+
+[Whisper mic](https://openai.com/blog/whisper/) Allows to use use a microphone continuously
```

Comment:
I assume you meant to link to https://github.com/mallorbc/whisper_mic?

Reply:
Ah, thanks! Wrong link from my copy and paste.

Reply:
I figured. I've made plenty of copy-paste mistakes myself :smile:.